File: test/fixtures/ruby/multiline_post_with_data.rb
require 'net/http'

uri = URI('http://localhost:28139/echo/html/')
req = Net::HTTP::Get.new(uri)
req.content_type = 'application/x-www-form-urlencoded'
req['Origin'] = 'http://fiddle.jshell.net'

req.set_form_data({
  'msg1' => 'value1',
  'msg2' => 'value2'
})

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end

File: test/fixtures/swift/multiline_post_with_data.swift
import Foundation

let data = NSMutableData(data: "msg1=value1".data(using: .utf8)!)
data.append("&msg2=value2".data(using: .utf8)!)

let url = URL(string: "http://localhost:28139/echo/html/")!
let headers = [
    "Origin": "http://fiddle.jshell.net",
    "Content-Type": "application/x-www-form-urlencoded"
]

var request = URLRequest(url: url)
request.allHTTPHeaderFields = headers
request.httpBody = data as Data

let task = URLSession.shared.dataTask(with: request) { (data, response, error) in
    if let error = error {
        print(error)
    } else if let data = data {
        let str = String(data: data, encoding: .utf8)
        print(str ?? "")
    }
}

task.resume()
